packages/pwa-kit-extension-sdk/src/global.d.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {SerializedExtension} from './types'
+
+declare global {
+    interface Window {
+        __EXTENSIONS__: {[key: string]: SerializedExtension}
```

Comment:
I've only added the properties we use in `pwa-kit-extension-sdk` as I didn't want to have to type all the fields. I'm thinking we can add the other fields on a case by case basis